pkg/operators/v1/olmconfig_types.go

```diff
@@ -23,6 +25,9 @@ type Features struct {
 	// When reenabled, OLM will recreate the "Copied CSVs" for each
 	// cluster scoped operator.
 	DisableCopiedCSVs *bool `json:"disableCopiedCSVs,omitempty"`
+	// PackageServerWakeupInterval is used to define the wakeup interval
+	// for packagerservers
```

Comment:
Would it be possible to make this more descriptive?

Reply:
Updated.